README.md

```diff
@@ -28,56 +28,30 @@ At time of writing, it also serves the priority campaign pages. See the [Campaig
 
 ## Technical documentation
 
-This is a public facing Ruby on Rails application that retrieves browse content from APIs and presents it.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
```

Comment:
Nothing I can see wrong with the new content, but I think that first line that mentions it being public facing was useful. And if it's not out of date the what it does too ("retrieves browse content from APIs and presents it").

Reply:
This should be covered by the main PR I have to rewrite the README, where I've tried to consolidate descriptive stuff like this at the start (in particular, [this line](https://github.com/alphagov/collections/pull/2362/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)). I've gone into more detail about the search aspect of "APIs", but I've dropped the "retrieves browse content from APIs" because that's too vague to be useful (which APIs??). Since all our frontend apps retrieve content from the Content Store, I think we should avoid repeating this in each repo.

In future, each frontend app could link to a single doc about their shared architecture, as that would be much more helpful than learning about it in each repo in isolation. Out-of-scope for now, though.